diff --git a/node_modules/chart.js/types/index.esm.d.ts b/node_modules/chart.js/types/index.esm.d.ts
index 3066472..4d238eb 100644
--- a/node_modules/chart.js/types/index.esm.d.ts
+++ b/node_modules/chart.js/types/index.esm.d.ts
@@ -216,7 +216,7 @@ export const LineController: ChartComponent & {
 
 export type ScatterControllerDatasetOptions = LineControllerDatasetOptions;
 
-export interface ScatterDataPoint {
+export type ScatterDataPoint = {
   x: number;
   y: number;
 }
